src/chains.jl

```diff
@@ -819,7 +837,8 @@ function _cat(::Val{3}, c1::Chains, args::Chains...)
     new_info = NamedTuple{tuple(nontime_props...)}(tuple([c1.info[n] for n in nontime_props]...))
     new_info = merge(new_info, (start_time = starts, stop_time = stops))
 
-    return Chains(value, missing, c1.name_map, new_info)
+    weights = UnitWeights{eltype(value)}(size(value, 1))
```

Comment:
The type should not be based on `value` (again, problems with `Missing`):
```suggestion
    weights = UnitWeights{Bool}(size(value, 1))
```